.github/workflows/main.yml

```
@@ -41,7 +41,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [16.x, 18.x]
```

Only testing karma with one version is enough.
```suggestion
        node-version: [18.x]
```